### DISCUSSION TITLE
Translate

### DISCUSSION THREAD

**Discussion by Teie** [diStyApps/seait#5] [Ideas]
Hi, I can translate into Portuguese and Spanish. How could I do that there in CONSTANTS.py?

**Comment by diStyApps** [comment]
Thank you very much, I would really love that, i will start to work on it after next update that should be today or tomorrow, and it will through a json files.

**Reply by Teie** [reply]
How do you want me to call you in tutorial video?
Some name, nickname?

**Reply by diStyApps** [reply]
Thank you very much, Just diSty.

**Reply by diStyApps** [reply]
Hey how you doing?  i have added json files of the localization all translated with chatGPT, fill free to correct the files.

**Comment by Teie** [comment]
Cool, send some smoke signal when you need it. See ya.

**Reply by diStyApps** [reply]
Let the fire begin!
